packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -179,9 +179,18 @@ export default class QuanticFacet extends LightningElement {
   renderedCallback() {
     initializeWithHeadless(this, this.engineId, this.initialize);
     this.input = this.template.querySelector('.facet__searchbox-input');
-    if (this.facetContentChanged) {
-      this.facetContentChanged = false;
-      this.setFocusToFirstFacetValue();
+
+    if (this.nextFocusedIndex !== undefined) {
+      this.nextFocusedIndex =
+        typeof this.nextFocusedIndex === 'number'
+          ? this.nextFocusedIndex
+          : this.values
+              .map((item) => item.value)
+              .indexOf(this.nextFocusedIndex);
+
+      if (this.nextFocusedIndex >= 0) {
+        this.setFocusToFacetValue(this.nextFocusedIndex);
+      }
```

Comment:
Yeah, this is still 'wip' since I was not sure it would be the implementation that makes most sense.

Reply:
This feels like a bit much to read, I think it would be better if you copied the FocusTarget util from Atomic (but in a LWC-friendly way, perhaps like a class with a `onRender` method).

Reply:
> This feels like a bit much to read, I think it would be better if you copied the FocusTarget util from Atomic (but in a LWC-friendly way, perhaps like a class with a `onRender` method).

Thanks for the advice, I am not sure we could use the focusTarget implementation from Atomic, the facets values would re-render value with same order but different HTML element, once one value is selected. Is there any other possible to find next focused HTML element without knowing its index within the list?